backend/src/server.ts

```diff
@@ -42,11 +42,13 @@ app.listen({ port: PORT, host: IP }, (err) => {
   // ca.crt is the default CA bundle provided by the service account for kubernetes
   // service-ca.crt is the CA bundle provided by the service account for kubernetes used by prometheus
   // odh-ca-bundle.crt and odh-trusted-ca-bundle.crt are the CA bundles provided by the ODH platform
+  // ca-bundle.crt is the default CA bundle used by the system
   const caPaths = [
     '/var/run/secrets/kubernetes.io/serviceaccount/ca.crt',
     '/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt',
     '/etc/pki/tls/certs/odh-ca-bundle.crt',
```

Comment:
@harshad16 @LaVLaS @VaishnaviHire in odh notebook controller in future, for setting the env variables value of SSL_CERT_FILE, REQUESTS_CA_BUNDLE and so on, we could do a similar approach as suggested by @LaVLaS.

i.e. check mounted-in new files from new configmap in the following order

`/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem`  (the system default of CentOS Stream 9, UBI8+, RHEL9 ..., will aways work and not be mucked up)
`/etc/pki/tls/certs/odh-trusted-ca-bundle.crt ` (here the one from the injection annotation configmap part. ie. ca-bundle.crt in the [original configmap](https://docs.openshift.com/container-platform/4.10/networking/configuring-a-custom-pki.html#certificate-injection-using-operators_configuring-a-custom-pki) that was configured by openshift cluster admins in line with the proxy config, what Landon calls proxy w/ self-signed certs or I call proxy with corporate CA non-public CAs. Will no be mucked up provided openshift admins do their job, which they do in our firm)
`/etc/pki/tls/certs/odh-ca-bundle.crt` the one from new configmap odh-ca-bundle.crt data section, i.e. self-signed certs given via [DataScienceClusterInitialization CR 
spec.trustedCABundle.customCAbundle](https://github.com/opendatahub-io/opendatahub-operator/pull/848#issue-2120714285) for dev purposes mostly

Reply:
I'm gonna resolve this conversation to allow the PR to be merged, we can open a discussion on slack/a github issue if we wanna continue this topic.

Thanks again @shalberd for this input, it's really appreciated.

Reply:
very good, yes, what Landon wanted to make sure that the Linux default CA bundle location is on top of the list .... which it is now. Nice